courses/free-courses-hi.md

```diff
@@ -742,16 +742,20 @@
 
 * [Complete Ruby Programming Course \|\| Hindi](https://www.youtube.com/playlist?list=PLgPJX9sVy92y_ENwi_4hMCpDtQFnazFPp) - CS Geeks
 * [Learn Ruby in 45 Minutes \| Hindi](https://www.youtube.com/watch?v=3V9a_WYEQPA) - CS Geeks
+* [Ruby Tutorial for Beginners (Hindi)](https://www.youtube.com/playlist?list=PLgPJX9sVy92yefe1xmyxgcyXjxmLHsSEV) - CS Geeks
 
 
 #### Ruby on Rails
 
 * [Full Stack Development in Ruby on Rails - Hindi](https://www.youtube.com/playlist?list=PLSfx1NJkuWPWlVjFy5datW4Y-54ltIFw7) - APPSIMPACT Academy Hindi
 * [Ruby on Rails 5 Tutorial(Hindi)](https://www.youtube.com/playlist?list=PLgPJX9sVy92yV7Qt6_8ElC9paGWdtdIbb) - CS Geeks
+* [Ruby on Rails Course \| Urdu / Hindi](https://www.youtube.com/playlist?list=PL9WbyKqkuCAYT-IPvo5PxR_hijMuR1RCB) - Code with Naqvi
```

Comment:
we have an Urdu list - this probably belongs there

Reply:
I understand Hindi and after seeing the videos in this playlist, its obvious to me that native Hindi speakers will be able to understand all the videos in the playlist. I don't speak Urdu so i am not sure if people who only speak Urdu would be able to understand, which is why I only kept it in the Hindi playlist. The main language used in the videos is surely Hindi.

Reply:
I think it should be listed in both Urdu and Hindi resource files

Reply:
I guess we need review from an urdu speaker.

Reply:
So should I add it in the Urdu file too?

Reply:
no we do not duplicate items across lists